cmd/sts-handlers.go

```diff
@@ -343,14 +343,21 @@ func (sts *stsAPIHandlers) AssumeRoleWithSSO(w http.ResponseWriter, r *http.Requ
 	// JWT custom claims.
 	var policyName string
 	policySet, ok := iampolicy.GetPoliciesFromClaims(m, iamPolicyClaimNameOpenID())
+	policies := strings.Join(policySet.ToSlice(), ",")
```

Comment:
Not sure why this was moved here @donatello?

Reply:
Moved? It's a new variable. Just to reuse the slicing and joining work.

Reply:
It's reused in the error message.